### PR TITLE
Update to latest logsearch-for-cloudfoundry

### PR DESCRIPTION
**Pull request by jmcarp**
Tasks remaining:
* [x] Restore hardening jobs
* [x] Run smoke test errand
* [x] Resize VMs

@LinuxBozo

### CODE REVIEW

File: logsearch-deployment.yml [hunk 1]
```diff
@@ -1,104 +1,100 @@
...
     # Default vcap & root password on deployed VMs (ie c1oudc0w)
     # Generated using mkpasswd -m sha-512
     bosh:
-      password: (( merge || "$6$4gDD3aV0rdqlrKC$2axHCxGKIObs6tAmMTqYCspcdvQXh3JJcvWOY2WGb4SrdXtnCyNaWlrf3WEqvYR2MYizEGp3kMmbpwBC6jsHt0" ))
-  stemcell: (( merge ))
-  disk_size:
-    elasticsearch_master: 50_000
-    elasticsearch_data: 500_000
-    queue: 75_000
+      password: "$6$4gDD3aV0rdqlrKC$2axHCxGKIObs6tAmMTqYCspcdvQXh3JJcvWOY2WGb4SrdXtnCyNaWlrf3WEqvYR2MYizEGp3kMmbpwBC6jsHt0"
```
Comment (LinuxBozo) [inline logsearch-deployment.yml]: This needs to be merged in..

Comment (jmcarp) [inline logsearch-deployment.yml]: I restored the usual `password: (( merge || "..." ))`, but spruce doesn't allow it--and the password will be merged from secrets anyway. If we want to ensure that the default password is never used, we could use `password: ~` instead.

Comment (LinuxBozo) [inline logsearch-deployment.yml]: I think the `password: ~` is a good solution. Should never use a default.

Comment (jmcarp) [inline logsearch-deployment.yml]: Done.

**Pull request merged**